extractor.js

```diff
@@ -0,0 +1,44 @@
+const { existsSync, mkdirSync, readFileSync, writeFileSync } = require('fs');
+const { i18nextToPot } = require('i18next-conv');
+const recursive = require("recursive-readdir");
+
+const argv = require('minimist')(process.argv.slice(2));
+const filename = argv['o'] || 'en.pot';
+
+const JS_FUNCTION_REGEX = new RegExp("i18next.t\\('(.*)'\\)", 'g');
```

Comment:
Should this perhaps also support `" "`(double quotes)

Reply:
It could be. It has to be decided for each application. I have talked with Nico and explained the idea.